locale/es/LC_MESSAGES/django.po

```diff
@@ -3282,892 +3023,853 @@ msgid ":?.!"
 
 #: venv/lib/python3.8/site-packages/django/forms/fields.py:613
 msgid "Upload a valid image. The file you uploaded was either not an image or a corrupted image."
-msgstr ""
+msgstr "Subir una imagen válida. El archivo que subiste o no fue imagen o fue corrupto."
 
 #: venv/lib/python3.8/site-packages/django/forms/fields.py:775
 #: venv/lib/python3.8/site-packages/django/forms/fields.py:865
 #: venv/lib/python3.8/site-packages/django/forms/models.py:1296
 #, python-format
 msgid "Select a valid choice. %(value)s is not one of the available choices."
-msgstr ""
+msgstr "Selecciona una opción válida. %(value)s no es una de las opciones disponibles."
 
 #: venv/lib/python3.8/site-packages/django/forms/fields.py:866
 #: venv/lib/python3.8/site-packages/django/forms/fields.py:981
 #: venv/lib/python3.8/site-packages/django/forms/models.py:1295
-#, fuzzy
-#| msgid "Email address:"
 msgid "Enter a list of values."
-msgstr "Dirección de correo electrónico:"
+msgstr "Ingrese una lista de valores."
 
 #: venv/lib/python3.8/site-packages/django/forms/fields.py:982
 msgid "Enter a complete value."
-msgstr ""
+msgstr "Ingresa un valor completo."
 
 #: venv/lib/python3.8/site-packages/django/forms/fields.py:1198
-#, fuzzy
-#| msgid "Email address:"
 msgid "Enter a valid UUID."
-msgstr "Dirección de correo electrónico:"
+msgstr "Ingrese una UUID válida."
 
 #: venv/lib/python3.8/site-packages/django/forms/fields.py:1228
-#, fuzzy
-#| msgid "Email address:"
 msgid "Enter a valid JSON."
-msgstr "Dirección de correo electrónico:"
+msgstr "Ingrese una JSON válida."
 
 #. Translators: This is the default suffix added to form field labels
 #: venv/lib/python3.8/site-packages/django/forms/forms.py:78
 msgid ":"
-msgstr ""
+msgstr ":"
 
 #: venv/lib/python3.8/site-packages/django/forms/forms.py:205
 #, python-format
 msgid "(Hidden field %(name)s) %(error)s"
-msgstr ""
+msgstr "(Campo oculto %(name)s) %(error)s"
 
 #: venv/lib/python3.8/site-packages/django/forms/formsets.py:93
 msgid "ManagementForm data is missing or has been tampered with"
-msgstr ""
+msgstr "Datos de ManagementForm está ausento o ha sido corrompido"
 
 #: venv/lib/python3.8/site-packages/django/forms/formsets.py:345
 #, python-format
 msgid "Please submit %d or fewer forms."
 msgid_plural "Please submit %d or fewer forms."
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "Por favor, enviar %d o menos formularios."
+msgstr[1] "Por favor, enviar %d o menos formularios."
 
 #: venv/lib/python3.8/site-packages/django/forms/formsets.py:352
 #, python-format
 msgid "Please submit %d or more forms."
 msgid_plural "Please submit %d or more forms."
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "Por favor, enviar %d o más formularios."
+msgstr[1] "Por favor, enviar %d o más formularios."
 
 #: venv/lib/python3.8/site-packages/django/forms/formsets.py:379
 #: venv/lib/python3.8/site-packages/django/forms/formsets.py:386
+#, fuzzy
+# TODO cc @mouse is this a verb or noun
```

Comment:
wait I have no clue! I don't think these should be in here, and I think it's because I put `venv3` in the ignore instead of `venv`.

https://github.com/bookwyrm-social/bookwyrm/blob/main/bw-dev#L93:
`runweb django-admin makemessages --no-wrap --ignore=venv3 $@`

Reply:
lol oops! well based on [this](https://github.com/django/django/blob/main/django/forms/formsets.py) i think it's a verb and also a completely different one, like order in the sense of arranging based on a rule, not order in the sense of you order food at a restaurant  

so i'd have to change it if it's to stay, but if i understand you right it really shouldn't be in there anyway?

Reply:
I think all these are translated internally in Django already 😬

Reply:
alright  i changed it to venv and then rebuilt